upgrade.md

```diff
@@ -17,6 +17,139 @@ The `storagePath` method of the `Illuminate\Contracts\Foundation\Application` in
+        );
+
+    // YOU NEED TO CHANGE IT TO
+    $email
+        ->from('taylor@laravel.com')
+        ->to('example@example.org')
+        ->subject("Order Shipped")
+        ->html('<h1>HTML email</h1><p>Shows HTML tags if the client supports it.</p>')
+        ->text('Some Plain text in the email');
+
+##### Removed `auth_mode` for SMTP
+Setting the auth_mode in the mail config was removed, because the authentication mode can be automatically negotiated between the Mailer and the SMTP server.
+
+##### Failed recipients
+It's no longer possible to get a list of failed recipients. Instead, if an email is failed to send, and exception will be thrown.
+
+##### Stream Options for SMTP transport
+Setting stream options for the SMTP transport can no longer be done through an smtp key. Instead you need to set the options directly in the config. For example, when disabling TLS peer verification in `config/mail.php`:
+
+    'smtp' => [
+
+        // BEFORE
+        'stream' => [
+            'ssl' => [
+
+                'verify_peer' => false,
+
+            ],
+        ],
+
+        // AFTER
+        'verify_peer' => false,
+    ],
+
+> {note} It is generally not advised to disable the Verification of SSL-Certificates because it introduces the possibiliets of Man-in-the-Middle attacks.
+
+For more options you can take a look at the [Symfony Mailer Docs](https://symfony.com/doc/6.0/mailer.html#transport-setup)
+
+##### Generated Messages ID's
+SwiftMailer offered the ability to set a custom domain to generate Message ID's through register it to mime.idgenerator.idright. This is no longer possible with Symfony Mailer. Instead, Symfony Mailer will automatically generate a Message ID based on the sender and add it to a message automatically.
+
+##### Removed Force Reconnection
+It is no longer possible to force a reconnection (for example when the mailer is running through a daemon process). Instead, Symfony Mailer will properly attempt to reconnect the transport itself and error out if that fails
+
+##### Driver / Transport Prerequisites
+`guzzlehttp/guzzle` can be removed if wanted (Still required to use with the HTTP Client and the ping methods on schedules). Symfony API mail transport will utilize `symfony/http` instead.
+
+###### SES
+`aws/aws-sdk-php` is not needed for SES Transport anymore and can be removed if wanted (Still required to use with the SQS queue driver and DynamoDb failed job storage). To continue using the SES Mailer you are required to use the underlying Symfony Amazon Mailer:
+
+    composer require symfony/amazon-mailer
+
+###### Mailgun
+To continue using the Mailgun Mailer you are required to use the underlying Symfony Mailgun Mailer:
+
+    composer require symfony/mailgun-mailer
+
+###### Postmark
+`wildbit/swiftmailer-postmark` needs to be removed because Laravel moved from Swift Mailer to Symfony Mailer. To continue using Postmark Mailer you are required to use the underlying Symfony Postmark Mailer:
```

Comment:
```suggestion
###### Postmark

`wildbit/swiftmailer-postmark` needs to be removed because Laravel moved from SwiftMailer to Symfony Mailer. To continue using Postmark Mailer you are required to use the underlying Symfony Postmark Mailer:
```